hera_xblocks/question/question/static/html/question.html

```diff
@@ -28,10 +27,46 @@
                             <option class="questions-list__select-item" value=""> ------- </option>
                             % for option in problem_type['options']:
                                 % if option.get('title'):
-                                    <option value="${ option['title'] }">${option['title']}</option>
+                                    <% selected = 'selected' if user_answer and option['title'] in user_answer[index] else '' %>
+                                    <option value="${option['title']}" ${selected}>${option['title']}</option>
                                 % endif
                             % endfor
                         </select>
+                    % elif problem_type['type'] == "table":
+                        <table>
+                            <tbody>
+                                % if problem_type['tableData'].get('columns'):
+                                    <tr>
+                                        % for column in problem_type['tableData']['columns']:
+                                            % if column['value'].startswith('!'):
+                                                <th>${column['value'].replace('!', '', 1)}</th>
+                                            % else:
+                                                <td>${column['value']}</td>
+                                            % endif
+                                        % endfor
+                                    </tr>
+                                % endif
+                                % if problem_type['tableData'].get('rows'):
+                                    <% user_answer_input_index = 0 %>
+                                    % for row in problem_type['tableData']['rows']:
+                                        <tr>
+                                            % for key in range(0, len(row)):
+                                                % if row[str(key)]['value'].startswith('?'):
+                                                    <td>
+                                                        <input type="text" name="${index}" value="${user_answer[index][user_answer_input_index] if user_answer else ''}"/>
+                                                    </td>
+                                                    <% user_answer_input_index += 1 %>
+                                                % elif row[str(key)]['type'] == 'head':
```

Comment:
please be consistent, use or `column['value'].startswith('!')` or `row[str(key)]['type'] == 'head'`

Reply:
Done